### ISSUE TITLE
Semantic segmentation Pretrained model?

### ISSUE THREAD

**Issue by wuyuanmm** [implus/UM-MAE#8]
I use upernet_mae_swin_tiny_256_mask...py, but when I use checkpoint-99-model.pth as the pretrained model. It report list:

mmseg - WARNING - The model and loaded state dict do not match exactly
unexpected key in source state_dict: fc_norm.weight, fc_norm.bias, head.weight, head.bias

when traing with this model, Gradient overflow.

**Comment by implus** [comment]
Hi, we do not observe the gradient problem when training upernet using the pre-trained model. The message is correct since segmentation model indeed doesnot need fc and head layers of the pre-trained classification model.
